dhis-2/dhis-api/src/main/java/org/hisp/dhis/scheduling/JobConfiguration.java

```diff
@@ -380,6 +380,13 @@ public boolean isRunOnce() {
         && queueName == null;
   }
 
+  public boolean isReadyToRun() {
+    return schedulingType == SchedulingType.ONCE_ASAP
+        || schedulingType == SchedulingType.CRON && cronExpression != null
```

Comment:
Just checking, do we want to use `StringUtils.notBlank` or some light validation of the cron expression to ensure it is valid?